features/step_definitions/appointment_locator_steps.rb

```diff
@@ -0,0 +1,14 @@
+When(/^I search for appointment locations near to a valid postcode$/) do
+  postcode = 'SW1A 2HQ'
+
+  allow(Locations).to receive(:near_postcode).with(postcode) do
+    1.upto(5).map { |x| Locations::Location.new("cab-#{x}") }
+  end
+
+  Pages::Locations.new.load(postcode: postcode)
+end
+
+Then(/^I should see the (\d+) appointment locations nearest to that postcode$/) do |_number_of_locations|
+  expected_locations = %w(cab-1 cab-2 cab-3 cab-4 cab-5)
```

Comment:
As we're ignoring `_number_of_locations` the convention would be to name it `_`, however, why are we ignoring it? At the moment it's a coincidence that `expected_locations` contains the same number of items we've used in our gherkin.

Maybe?

```rb
expected_locations = number_of_locations.times.map { |i| "cab-#{i+1}" }
```